.travis.yml

```diff
@@ -17,9 +17,9 @@ matrix:
   # to whatever the latest default is.
   include:
     - python: 2.7
-      env: "CRYPTO_BEFORE=1.6"
```

Comment:
IIRC this silly bit here (and the matching one just below) was because of some sort of wacky problem differing between Cryptography 1.5 and 1.6 (since it seems to force installation of, effectively, Cryptography 1.5).

Somebody (me or other) needs to look up the commits from when we added this & confirm whether or not it is still valid. If we're taking Crypto >=2.5 as the new dependency, some Crypto 1.5/1.6 issue isn't going to matter going forwards, and we could nix this entire test component.

Reply:
Presumabely the point is, going forward, to ensure that as we release new cryptography versions, the paramiko doesn't accidentally start relying on newer features without changing setup.py

Reply:
IIRC my pondering was specifically about "what changed in crypto 1.6 that made it some sort of threshold to cross?". At this point in time I'm not sure how hard I care anymore though. Not going to be applying this to all my old branches so their travis.ci's will continue to give this shit & the more recent branches will probably stop doing so.